src/containers/tutor-home-page/general-info/GeneralInfo.js

```diff
@@ -1,71 +1,163 @@
+import { useEffect, useState, useCallback } from 'react'
 import { useTranslation } from 'react-i18next'
+import { LocationService } from '~/services/location-service'
 
+import { createFilterOptions } from '@mui/material'
 import Box from '@mui/material/Box'
 import Typography from '@mui/material/Typography'
 import Checkbox from '@mui/material/Checkbox'
 import FormControlLabel from '@mui/material/FormControlLabel'
 
 import AppTextField from '~/components/app-text-field/AppTextField'
+import AppAutoComplete from '~/components/app-auto-complete/AppAutoComplete'
+import useBreakpoints from '~/hooks/use-breakpoints'
+import useAxios from '~/hooks/use-axios'
+import useForm from '~/hooks/use-form'
 
-import img from '~/assets/img/tutor-home-page/become-tutor/general-info.png'
+import img from '~/assets/img/tutor-home-page/become-tutor/general-info.svg'
+import { useStepContext } from '~/context/step-context'
+import { validations } from '~/containers/tutor-home-page/constants'
 import { styles } from '~/containers/tutor-home-page/general-info/general-info.styles'
 
-const GeneralInfo = ({ data, handleChange, handleBlur, errors, btnsBox, stepLabel }) => {
+const GeneralInfo = ({ btnsBox, stepLabel }) => {
   const { t } = useTranslation()
+  const { isDesktop, isMobile } = useBreakpoints()
+  const [counterColor, setCounterColor] = useState('#263238')
```

Comment:
[error, setError] =useState(false) and move color to palette

Reply:
fixed